corehq/apps/reports/standard/cases/data_sources.py

```diff
@@ -176,7 +178,17 @@ def _get_username(self, user_id):
         return username
 
     def parse_date(self, date_string):
-        return string_to_utc_datetime(date_string)
+        try:
+            return string_to_utc_datetime(date_string)
+        except:
+            try:
```

Comment:
@snopoke @benrudolph I took a first pass at writing a simple-to-use tool for this use-case here: https://github.com/dimagi/commcare-hq/pull/6126. For now I'm going the route of emailing specified individuals directly, especially as I debug this, but if that ends up being too limiting, we can revisit it.